Communication/Protocols/UCI/Options/StringOption.cs

```diff
@@ -0,0 +1,28 @@
+namespace Communication.Protocols.UCI
+{
+    class StringOption : BaseOption
+    {
+        private static string EmptyValue = "<empty>";
```

Comment:
You have started all field names before with lower case, but this with upper case. Why?

Reply:
I was planning to make it `const`. I forgot to add the modifier.